netty/src/main/java/io/grpc/netty/NettyServerHandler.java

```diff
@@ -211,7 +222,8 @@ private void onHeadersRead(ChannelHandlerContext ctx, int streamId, Http2Headers
     }
   }
 
-  private void onDataRead(int streamId, ByteBuf data, boolean endOfStream) throws Http2Exception {
+  private void onDataRead(int streamId, ByteBuf data, int padding, boolean endOfStream) throws Http2Exception {
+    dataSinceLastPing += data.readableBytes() + padding;
```

Comment:
data is overloaded here.  I assume data means a set of bytes, rather than the number of bytes.   Perhaps rename this to dataSizeSincePing?